src/tests/eventpipe/ThreadPoolEvents.cs

```diff
@@ -0,0 +1,168 @@
+                    //source.Clr.ThreadPoolWorkerThreadStop += (eventData) => ThreadStopEvents += 1;
+
+                    int ThreadPoolWorkerThreadAdjustmentSampleEvents = 0;
+                    int ThreadPoolWorkerThreadAdjustmentAdjustmentEvents = 0;
+                    source.Clr.ThreadPoolWorkerThreadAdjustmentSample += (eventData) => ThreadPoolWorkerThreadAdjustmentSampleEvents += 1;
+                    source.Clr.ThreadPoolWorkerThreadAdjustmentAdjustment += (eventData) => ThreadPoolWorkerThreadAdjustmentAdjustmentEvents += 1;
+
+                    return () => {
+                        Logger.logger.Log("Event counts validation");
+
+                        Logger.logger.Log("ThreadStartEvents: " + ThreadStartEvents);
+                        //Logger.logger.Log("ThreadStopEvents: " + ThreadStopEvents);
+                        //bool ThreadStartStopResult = ThreadStartEvents >= 1 && ThreadStopEvents >= 1;
+                        bool ThreadStartStopResult = ThreadStartEvents >= 1;
+                        Logger.logger.Log("ThreadStartStopResult check: " + ThreadStartStopResult);
+
+                        Logger.logger.Log("ThreadPoolWorkerThreadAdjustmentSampleEvents: " + ThreadPoolWorkerThreadAdjustmentSampleEvents);
+                        Logger.logger.Log("ThreadPoolWorkerThreadAdjustmentAdjustmentEvents: " + ThreadPoolWorkerThreadAdjustmentAdjustmentEvents);
+                        bool ThreadAdjustmentResult = ThreadPoolWorkerThreadAdjustmentSampleEvents >= 1 && ThreadPoolWorkerThreadAdjustmentAdjustmentEvents >= 1;
+                        Logger.logger.Log("ThreadAdjustmentResult check: " + ThreadAdjustmentResult);
+                        
+                        return ThreadStartStopResult && ThreadAdjustmentResult ? 100 : -1;
+                    };
+                };
+
+                var config = new SessionConfiguration(circularBufferSizeMB: (uint)Math.Pow(2, 10), format: EventPipeSerializationFormat.NetTrace,  providers: providers);
+
+                var ret = IpcTraceTest.RunAndValidateEventCounts(_expectedEventCounts, _eventGeneratingAction, config, _DoesTraceContainEvents);
+                Assert.Equal(100, ret);
+            }, output);
+        }
+        [Fact]
+        public async void IOThread_ProducesEvents()
+        {
+            await RemoteTestExecutorHelper.RunTestCaseAsync(() => 
+            {
+                Dictionary<string, ExpectedEventCount> _expectedEventCounts = new Dictionary<string, ExpectedEventCount>()
+                {
+                    { "Microsoft-Windows-DotNETRuntime", -1 },
+                    { "Microsoft-Windows-DotNETRuntimeRundown", -1 },
+                    { "Microsoft-DotNETCore-SampleProfiler", -1 }
+                };
+
+                var providers = new List<Provider>()
+                {
+                    new Provider("Microsoft-DotNETCore-SampleProfiler"),
+                    //GCKeyword (0x1): 0b1
```

Comment:
Comment doesn't seem to match the code?

Reply:
Fixed: the comment is belong to IOThread test, this test has been disabled.